src/app/pages/administration/email-templates/email-templates.component.ts

```diff
@@ -65,14 +66,16 @@ export class EmailTemplatesComponent implements OnInit {
 
   public onFilter(filter: Filter): void {
     this.emailTemplatesRequestOptions.filter = filter;
+    this.getEmailTemplates();
```

Comment:
Why we need this?

Reply:
It is to implement filter functionality for the email templates, just like with the assessments. Once we add some filters via the side drawer, we need to make another API call to update the template with the filtered data.